pkg/clustermanager/cluster_manager.go

```diff
@@ -1395,3 +1395,19 @@ func (c *ClusterManager) buildSpecForCluster(ctx context.Context, clus *types.Cl
 func (c *ClusterManager) DeletePackageResources(ctx context.Context, managementCluster *types.Cluster, clusterName string) error {
 	return c.clusterClient.DeletePackageResources(ctx, managementCluster, clusterName)
 }
+
+// UpdateEKSAManagementComponentVersion updates the eksa-management-component-version annotation for the managment cluster.
+func (c *ClusterManager) UpdateEKSAManagementComponentVersion(ctx context.Context, managementCluster *types.Cluster, clusterSpec *cluster.Spec) error {
```

Comment:
Update*Annotation

Reply:
Do you mean `UpdateEKSAManagementComponentVersionAnnotation`?

Reply:
UpdateManagementComponent**s**VersionAnnotation, if you changed the const